include/device.h

```diff
@@ -323,9 +369,6 @@ struct device_pm {
 	struct k_poll_signal signal;
 };
 
-/**
- * @brief Runtime device structure (in memory) per driver instance
- */
```

Comment:
?

Reply:
The original comment was wrong (this isn't mutable as implied), restored and corrected.